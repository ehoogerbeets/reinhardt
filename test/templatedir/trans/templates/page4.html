{% blocktrans comment "This is the translator's comment" %}
This is the fourth block trans with multiple
lines in the source and a translator's comment which should not appear 
here in the output.
{% endblocktrans %}
