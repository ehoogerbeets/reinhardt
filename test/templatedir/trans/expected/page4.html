 Dies ist der vierte trans-Block mit mehreren Zeilen in der Quelle und dem Kommentar des Ãœbersetzers, die in der Ausgabe nicht erscheinen soll. 
